core/src/test/java/bio/terra/pearl/core/service/kit/pepper/LivePepperDSMClientTest.java

```diff
@@ -65,7 +70,11 @@ void startup(TestInfo info) throws Exception {
 
     @Transactional
     @Test
-    public void testBadErrorResponseFromPepper() throws Exception {
+    public void testBadErrorResponseFromPepper(TestInfo info) throws Exception {
+        Enrollee enrollee = enrolleeFactory.buildPersisted(info.getDisplayName());
```

Comment:
we've got a helper for consistency here
```suggestion
        Enrollee enrollee = enrolleeFactory.buildPersisted(getTestName(info));
```

Reply:
and you'll want to update the others too